Sources/PixelKit/PixelKit.swift

```diff
@@ -54,6 +54,11 @@ public final class PixelKit {
         /// This is useful in situations where pixels receive spikes in volume, as the daily pixel can be used to determine how many users are actually affected.
         case dailyAndCount
 
+        /// Sent once per day with a `_daily` suffix, in addition to every time it is called with the default name (no suffix).
+        /// This means a pixel will get sent twice the first time it is called per-day, and subsequent calls that day will only send the pixel with a standard name.
+        /// This is useful in situations where pixels receive spikes in volume, as the daily pixel can be used to determine how many users are actually affected.
+        case dailyAndStandard
```

Comment:
Contrary to `dailyAndCount`, this will fire a daily pixel (with `_daily` suffix) and the standard pixel with an unchanged name. It's to preserve the original pixel name while adding a daily variant.